### PR TITLE
Add recipe for samsungtvws

### PR DESCRIPTION
**Pull request by mlondschien**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [ ]~~ Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mlondschien** [comment]
@xchwarze I would like to add samsungtvws to conda-forge. This would simplify installing it for some users. Are you OK with being added as a maintainer of the conda package? A simple 👍 suffices.

**Comment by xchwarze** [comment]
Ok!
I think you should add mlondschien as well.

File: recipes/samsungtvws/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00508dd5bf34199db8837a167b5aaa73bdb2590cb8fbfb28acbf33c0e60ceefa
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/samsungtvws/meta.yaml:18]:
```suggestion
    - python >=3.6
```

File: recipes/samsungtvws/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/samsungtvws/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/samsungtvws/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00508dd5bf34199db8837a167b5aaa73bdb2590cb8fbfb28acbf33c0e60ceefa
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/samsungtvws/meta.yaml:14]:
```suggestion
  number: 0
  noarch: python
```

**Comment by mlondschien** [comment]
Thanks, @marcelotrevisani and @xchwarze. @conda-forge/staged-recipes, this is ready for review.

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**